plugins/members-area-banking/controllers/banking.coffee

```diff
@@ -57,7 +57,7 @@ class BankingController extends Controller
   _reprocess: (done) ->
     @req.models.Transaction.find().run (err, transactions) =>
       return done err if err
-      regex = /^(.*) M(0[0-9]+) ([A-Z]{3})$/
+      regex = /^(.*) M(?:O|0)*(\d*) ([A-Z]{3})$/
```

Comment:
I'll need to go through the old transactions and see what additional hits the new regex gives before committing to it - don't want any false positives!